package.json

```diff
@@ -51,6 +53,8 @@
     "tsc": "tsc",
     "format-php": "composer format:all",
     "phpstan": "composer phpstan",
+    "test-e2e": "wp-scripts test-playwright --config plugins/auto-sizes/tests/e2e/playwright.config.ts",
+    "test-e2e:debug": "wp-scripts test-playwright --config plugins/auto-sizes/tests/e2e/playwright.config.ts --ui",
```

Comment:
What if we ever want to have e2e tests in other plugins too? It would be weird to copy these config files everywhere, so I think it would be better to move them to `tools/` or so. But then again see my other comments about their purpose.

Reply:
Update the infrastructure in https://github.com/WordPress/performance/pull/1988/commits/9fa9829a4c93437275a25a91e40d00b4584945fb